src/components/PressableWithSecondaryInteraction/index.native.js

```diff
@@ -1,6 +1,7 @@
 import _ from 'underscore';
 import React, {forwardRef} from 'react';
 import {Pressable} from 'react-native';
+import {LongPressGestureHandler, State} from 'react-native-gesture-handler';
```

Comment:
Just a thought. Do you think importing Pressable directly from `react-native-gesture-handler` would be enough? Could you please test it?

Reply:
react-native-gesture-handler not have Pressable . 

Also If it have.  It will have evevnts  with some different keys and will med map all events .
Also we cann't use it as ref (it get crashed with me ) . 
So it will not be recommended if it's not add value .

Reply:
Ok.